pkgs/development/python-modules/python-imports-check-libstdcxx/test_forward_compatibility_fails.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
i think a module docstring could help to explain what this script does and why it's important